kanopi/kanopi-pack
# Adjust JS import module resolution

## Description
> As a developer, I need to have Typescript modules resolved without extension, like JS files when using import statements. This avoids overriding standard ESLint rules.

- Specify and allow extra JS file import extension resolution
- Defaults to all .js, .ts, and .json files
- Add Kanopi Pack config settting `scripts.additionalResolveExtensions` and update documentation
- Update dependencies 

## Steps to Validate
1. Update the package in a supported project
1. Test current compilation, it should work as expected (considering dependency updates)
1. Add a tsconfig.json and TS entry point.
2. Add the entry point and a secondary TS file. Import the TS file without extension and confirm the import works and the script is built (you can test either the built or WDS version)
